Site/SiteMultipartMailMessage.php

```diff
@@ -337,6 +340,46 @@ public function addBcc($email)
 	}
 
 	// }}}
+	// {{{ public function getCcList()
+
+	/**
+	 * Gets an array of email addresses for CC
+	 *
+	 * @return arrays $email addresses for CC.
+	 */
+	public function getCcList()
+	{
+		$list = array();
+		foreach ($this->cc_list as $email) {
+			if (SwatString::validateEmailAddress(trim($email))) {
+				$list[] = trim($email);
+			}
+		}
+
+		return $list;
+	}
+
+	// }}}
+	// {{{ public function getBccList()
+
+	/**
+	 * Gets an array of email addresses for BCC
+	 *
+	 * @return arrays $email addresses for BCC.
+	 */
+	public function getBccList()
+	{
+		$list = array();
+		foreach ($this->bcc_list as $email) {
+			if (SwatString::validateEmailAddress(trim($email))) {
```

Comment:
See [this issue](https://github.com/silverorange/site/pull/181/files#r68611272).